client/settings/express-checkout/apple-google-pay-item.tsx

```diff
@@ -69,10 +69,10 @@ const AppleGooglePayExpressCheckoutItem = (): React.ReactElement => {
 										methodsConfiguration.apple_pay
 											.description
 									}
-									{ isPaymentRequestEnabled && ' ' }
+									{ ! isPaymentRequestEnabled && ' ' }
 									{
 										/* eslint-disable jsx-a11y/anchor-has-content */
-										isPaymentRequestEnabled &&
+										! isPaymentRequestEnabled &&
```

Comment:
Is there a reason for this change? I don't see it in the related issues.

FWIW I don't see a reason to toggle the visibility of those statements, we could just leave them on regardless.